<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayerTreeGroupPropertiesWidgetBase</class>
 <widget class="QWidget" name="QgsLayerTreeGroupPropertiesWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Group Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="mRenderAsGroupCheckBox">
     <property name="title">
      <string>Render Layers as a Group</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QgsBlendModeComboBox" name="mBlendModeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblTransparency">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lblBlend">
        <property name="text">
         <string>Blending mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QgsOpacityWidget" name="mOpacityWidget" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QgsEffectStackCompactWidget" name="mEffectWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsBlendModeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsblendmodecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsEffectStackCompactWidget</class>
   <extends>QWidget</extends>
   <header>qgseffectstackpropertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsOpacityWidget</class>
   <extends>QWidget</extends>
   <header>qgsopacitywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
  <tabstops>
  <tabstop>mRenderAsGroupCheckBox</tabstop>
  <tabstop>mOpacityWidget</tabstop>
  <tabstop>mBlendModeComboBox</tabstop>
  <tabstop>mEffectWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
